window-placement.lisp

```diff
@@ -28,33 +28,61 @@
 (defun string-match (string pat)
   (ppcre:scan (get-or-create-rule-scanner pat) string))
 
-(defun window-matches-properties-p (window &key class instance type role title)
+
```

Comment:
You added an extra space here.

Reply:
Thanks for catching that, not sure how i missed it